### ISSUE TITLE
Create Table: accomplishment_type

### ISSUE THREAD

**Issue by ericvennemeyer** [hackforla/peopledepot#61]
### Overview

We need to create the accomplishment_type table so that we can update a shared data store across hackforla.org, vrms, civictechjobs, and tables (onboarding) project.

#### Details

A table and a model are the same thing

### Action Items

- [ ] identify and document table description (see spreadsheet under Resources)
  - [ ] if not, reach out to PD leads
- [ ] compare and check off the data fields (below) against the ERD. Note any fields in the ERD that's missing from the list. (see Resources)
- [ ] compare the check off associated tables (below) against the ERD. Note any associated tables in the ERD that's missing from the list (see Resources)
- [ ] create a single model in Django (defining schema)
- [ ] write a test for the relationships this model will have with other models (e.g., creating a user and assigning them a set of permissions on a project).
- [ ] write an API end point
- [ ] write API unit tests
- [ ] document the endpoint

### Resources/Instructions

- See [People Depot Resources wiki page](https://github.com/hackforla/peopledepot/wiki/Resources-and-Links) for links
  - ERD
  - Table and Field Definitions
  - API Endpoint
- See the [wiki page](https://github.com/hackforla/peopledepot/wiki/Create-Table-issues-data-gathering-workflow) for instructions.

### Description

Dictionary of accomplishments to use in the wins table

### Data Fields

1. Copied from spreadsheet and checked off according to ERD. (unchecked items indicate a mismatch between ERD and spreadsheet, which requires a review)

    - [ ] (PK) id - int - Record Id
    - [ ] name - varchar
    - [ ] display_text - varchar
    - [ ] created - timestamp
    - [ ] updated - timestamp

1. In ERD only (having items here indicates a mismatch, which requires a review)
   - None

### Associated Tables

1. Copied from spreadsheet and checked off according to ERD. (unchecked items indicate a mismatch between ERD and spreadsheet, which requires a review)
   - [ ] win (one-to-many)

1. In ERD only (having items here indicates a mismatch, which requires a review)
   - None

**Comment by ExperimentsInHonesty** [comment]
This needs to be changed to win_type

**Comment by Neecolaa** [comment]
accomplishment_type has been renamed to win_type on the ERD, as well as the fields, tables, and data tabs of the spreadsheet

**Comment by fyliu** [comment]
@Neecolaa I still see an `accomplishment` table issue #48. But also a `win` table issue #82. Can you clarify what accomplishment and win are?

I think win is for individual accomplishment, created by an individual volunteer as an announcement of what they achieved.

**Comment by Neecolaa** [comment]
@fyliu You are correct, wins are for individual volunteers. Accomplishments are for projects.  `accomplishment_type` was renamed to `win_type` because it's meant to be for wins rather than accomplishments. 

The `accomplishment` table is still useful, but the project accomplishments are varied enough that they don't have a corresponding type table.

**Comment by fyliu** [comment]
Thanks! I'll prioritize the accomplishment table issue then since it's still needed.

**Comment by shmonks** [comment]
This issue needs to be cleaned up to resolve all comments and remove any info needed within the comments into the top part of the issue. Good first issues need to be ready to work.

**Comment by fyliu** [comment]
Comments are resolved and hidden. This is ready for prioritization.
